src/plugins/analysis/users_and_passwords/code/password_file_analyzer.py

```diff
@@ -9,51 +9,47 @@
 
 from analysis.PluginBase import AnalysisBasePlugin
 from helperFunctions.fileSystem import get_src_dir
+from helperFunctions.tag import TagColor
 
 
 class AnalysisPlugin(AnalysisBasePlugin):
     '''
-    This Plugin trys to find and crack passwords
+    This plug-in tries to find and crack passwords
     '''
     NAME = 'users_and_passwords'
     DEPENDENCIES = []
     MIME_BLACKLIST = ['audio', 'filesystem', 'image', 'video']
     DESCRIPTION = 'search for UNIX and httpd password files, parse them and try to crack the passwords'
-    VERSION = '0.4.1'
```

Comment:
Hm. Okay.

Reply:
there was a never published minor version in between ;-)